app/cryptography/jwt_encoder.py

```diff
@@ -27,28 +28,22 @@ def __init__(self):
     def encode(self, payload):
         return jwt.encode(payload, self.private_key, algorithm="RS256", headers={'kid': KID, 'typ': 'jwt'})
 
-    def encrypt_token(self, text, jwe_protected_header=None, cek=None, iv=None, encrypted_key=None, tag=None):
+    def encrypt_token(self, text, jwe_protected_header=None, encrypted_key=None, tag=None):
         """
         Overloading this method for test purposes, this allows us to modify the encrypted contents to test
         the decoder.
         :param text: the text to encrypt
         :param jwe_protected_header: the protected header, providing this overrides the internal one
-        :param cek: key for encryption, providing this overrides the generated internal one
-        :param iv: iv for encryption, providing this overrides the generated internal ones
         :param encrypted_key: the encrypted cek, providing this overrides the encrypted internal one
         :param tag: the authentication tag, providing this overrides the generated one
         :return: an encoded and encrypted text
         """
         if jwe_protected_header is None:
             jwe_protected_header = self._jwe_protected_header()
-        if cek is None:
-            cek = self.cek
-        if iv is None:
-            iv = self.iv
         if encrypted_key is None:
-            encrypted_key = self._encrypted_key(cek)
+            encrypted_key = self._encrypted_key(self.cek)
```

Comment:
We used them as args only for tests, we could have just set self.cek like we do now to change it for testing hence I they were removed